src/tokens-dist/json/css-variables-nested.json

```diff
@@ -645,6 +645,14 @@
           },
           "label": "var(--eds-theme-color-form-label)"
         },
+        "modal": {
+          "brand": {
+            "header": {
+              "background": {}
```

Comment:
Oof created ticket to fix this https://czi-tech.atlassian.net/browse/EDS-743?atlOrigin=eyJpIjoiNDE5MGExMjMyYjNhNGE3NGEwMjNmYjcwNDQ3MjVhN2IiLCJwIjoiaiJ9

Reply:
Thanks!

Reply:
@dierat I think we can continue with merging since I don't think not having this token in TW would be that blocking

Reply:
Awesome, thanks!